### DISCUSSION TITLE
502 Server Error - Acmebot not working anymore

### DISCUSSION THREAD

**Discussion by the-project-group** [polymind-inc/acmebot#715] [Q&A]
**Describe the bug**
Acmbeot was running for years, flawlessly ! Thanks for that!

Today I noticed that some certs expire soon and I was wondering, why they won't get renewed.

**To Reproduce**
Open the URL of the function app: https://my-function-app.azurewebsites.net/dashboard
It takes a while and then I just get this:
```
502 - Web server received an invalid response while acting as a gateway or proxy server.
There is a problem with the page you are looking for, and it cannot be displayed. When the Web server (while acting as a gateway or proxy) contacted the upstream content server, it received an invalid response from the content server.
```
![CleanShot 2024-06-04 at 18 14 28@2x](https://github.com/shibayan/keyvault-acmebot/assets/127997549/b22166df-aba9-4272-bbcf-b1d0032aab44)

Stopping / starting the function app doesn't help.

https://shibayan.blob.core.windows.net/azure-keyvault-letsencrypt/v3/latest.zip is throwing a "ResourceNotFound" error.

Any hints what I can check?
Thanks a lot and best regards, Flo.

**Comment by shibayan** [comment]
Acmebot v3 is no longer supported at the same time as the end of support for Azure Functions v3. To continue using it, you will need to upgrade to v4.

https://github.com/shibayan/keyvault-acmebot/discussions/500
